contracts/child/ChildChain.sol

```diff
@@ -71,12 +78,25 @@ contract ChildChain is Ownable {
     isERC721[rootToken] = isErc721;
   }
 
+  function onStateReceive(
+    bytes memory data
+  ) public onlyStateSyncer {
+    // address(this).call(abi.encodePacked(bytes4(keccak256("depositTokens(address,address,uint256,uint256)")), data));
```

Comment:
Is this comment obsolete now? Doesn't look like we are using this method anymore.